src/main/java/com/minecolonies/coremod/network/messages/PermissionsMessage.java

```diff
@@ -45,7 +45,7 @@
     /**
      * Client side presentation of the message.
      */
-    public static class View implements IMessage, IMessageHandler<View, IMessage>
+    public static class View extends AbstractMessage<View, IMessage>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 32 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)